### DISCUSSION TITLE
Where to post what

### DISCUSSION THREAD

**Discussion by supermerill** [supermerill/SuperSlicer#1765] [notice]
As there is currently an overlap over discussion and issues, here is a little guide on when to use an issue and when to open a discussion.
An **issue** (second tab, next to code) is good when:
 * A bug exists and you can share how to reproduce it.
 * A problem exists and you can explain why and how it could be resolved.
 * A feature is missing, and you can describe it with a good explanation. Why it's needed, problems if may resolve.
 * A feature is lacking and may be improved.

A **discussion** (this tab) is good when:
 * It's something you may want to post on discord / irc, but it's better here as it won't be lost after a day.
 * To ask how to do something (macro, modifiers, etc)
 * To discuss how to move forward with a feature (how to integrate non-planar layer, what to use) **when the issue is already created** (to not pollute the issue).
 * To synchronize your effort about something with other users (translation, doc, ...)
 * To share something, like a good picture of a printed object that could be used for the splashscreen.

**Comment by fmjensen** [comment]
So there is a Discord server 😮 this leaves me wondering if I am blind since I cannot find a link to it or maybe some kind of ritual is needed to be granted access 🙀 🤯

**Reply by fmjensen** [reply]
Okeh - I found [a link to it](https://discord.gg/4TVUG4VRzP) and might as well conclude that i am sort of blind 😬

**Comment by jackjrp** [comment]
what version of SS do I need to run on MacVersion 13.1.2 (13609.3.5.1.5)high Sierra Mini Mac please include a link I can't find my way around this website. Thank you for your help

**Comment by wscrivens** [comment]
I downloaded the Mac DMG file to my MacBook Pro running Ventura.  After getting past the Apple "protection" stuff, I tried running the app, and the icon appeared in the dock for an instant then disappeared.
I decided to go ahead and build it so I downloaded the latest source and ran the build script.  After nearly an hour to build, I opened the app from the generated DMG, and so far it's working perfectly!  Thanks to all who work so hard on this project!

**Comment by oliverhbailey** [comment]
Issues with Klipper and SuperSlicer -
It looks like this thread suffers from a lack of interest in developers either a) adding more topics with some real clarity on where items belong or b) a general waste basket of issues that no one cares about.

There are some issues with Klipper and SuperSlicer regarding tuning pressure advance.  When using the Klipper documentation, it appears pretty straight forward on how to configure a "slicer".  But SuperSlicer simply doesn't allow settings that go low enough for the line height.  Using a 0.04 nozzle, 0.03 would be the minimum line height but, setting this value is below the minimum line height allowed in the current version of SuperSlicer.  After reading through the documentation there are two tuning scenarios presented, a "direct drive" and a "long Bowden tube drive".  What defines a long Bowden tube drive?  Since a Bowden tube should be related to the maximum travel distance, this would appear to be aimed at machines with bigger build surfaces!
The problem here is that it does not appear the pressure advance is off by much but rather that the range of pressure changes may be either too high in each increment or is not designed to properly calibrate a smaller change to pressure advance using these techniques.

The problem I am having is simple,  when running a pressure advance sample,  the first layer goes down well but but the middle of the second layer the extruder stops working die to the amount of distance in pushing and pulling the filament.  I have run the filament feed tests and the filament is feeding 100 mm perfectly.  This is happening on a CR6-SE with an all metal extruder and Inland PLA+ filament.  This is a brand new spool o filament and I have gotten good parts off of it already, but attempting to tune using the Klipper documentation seems to be impossible.

I notice a lot of people who seem to have issues with Klipper and SuperSlicer.  I have been using Klipper and SuperSlicer for over a year with very little problems.  The bed on my machine is level to 0.136 and has not changed at all. But attempting to get a handle on this seems evasive at best, impossible at worst, and no one seems to know either of these tools well enough to offer a response on any fourm at all.  Everyone who comments seems to go ignored.  And the posting rules seem excessively rigid.

If anyone can answer from SuperSlicer or Klipper it would be greatly appreciated.

 Thank you in advance.

**Reply by anonymousaga** [reply]
I understand your perspective and the forum's perspectives. I don't mean to belittle or mock you, so please don't take this the wrong way, but some of the questions you asked are considered "common knowledge" in forums like Klipper.

> Using a 0.04 nozzle, 0.03 would be the minimum line height

0.04mm nozzles do not exist. Your nozzle is a 0.4mm nozzle. You can use layer heights between 0.08mm and 0.32mm by default SuperSlicer settings. I assume by "line height" you mean layer height, or how thick each layer is.
<br>

> What defines a long Bowden tube drive?

There are two types of printers: Bowden and Direct Drive.
"Long Bowden" printers have the extruder, or the motor that pushes the plastic filament, away from the hotend/nozzle. Such printers use a tube, called a bowden, to direct the filament into the hotend.
"Direct drive" (a.k.a. short bowden) printers have the extruder mounted right next to the hotend. 
Bowden/Direct drive is not related to the bed size. For example, the Prusa Mini is bowden, while the Prusa XL is direct drive.
Here is a video with basic terminology: https://youtu.be/ybTbuUBy2-s
<br>
> The problem here is that it does not appear the pressure advance is off by much but rather that the range of pressure changes may be either too high in each increment or is not designed to properly calibrate a smaller change to pressure advance using these techniques.

I am not able to understand what you are saying.
<br>
> The problem I am having is simple, when running a pressure advance sample, the first layer goes down well but but the middle of the second layer the extruder stops working die to the amount of distance in pushing and pulling the filament.

I hypothesize that your layer height is too low. Again, ideal layer height is 0.2-0.3mm, since your nozzle is 0.4mm
<br>
> But attempting to get a handle on this seems evasive at best, impossible at worst, and no one seems to know either of these tools well enough to offer a response on any fourm at all. Everyone who comments seems to go ignored. And the posting rules seem excessively rigid.

You are expected to understand your printer and its software with proper vocabulary on technical forums such as Klipper forums. There are great YouTube channels, such as Teaching Tech, who make tutorials on all things 3d printing and Klipper. Do some research and supplement the Klipper official documentation with Youtube videos and blog tutorials for beginners. Good luck with your future printing endeavors :)

**Reply by oliverhbailey** [reply]
You need to focus on medical school and keep your comments to yourself on things you know nothing about.  What makes you entitled to respond to anything I say?  Do you have a life or anything important to do.  I re-read your comments and its clear you are spouting your mouth like every other child in the late 20's to mid 30's.  Get a life and shut your mouth.  I raised four children and you would not make a pimple on their face.  And the oldest is 48 years old and doing very well, thank you.  Vocabulary.  I wrote some of the vocabulary in the personal computer dictionary back in the 1970's.  You would do well to shut your mouth on things you have no experience at all.  And if think that a vocabulary is going to make you a good doctor, you would be very wrong.  Get off these boards and focus on what you want to do in life, and grow a pair so you can play with the big boys some day.  Now go back to the hole you came from and find something important to do.

**Comment by oliverhbailey** [comment]
Thanks for your "better late than never reply".  The good news is that it
took you so long to read the original post, I forgot that I wrote it and
went on to solve all the problems and improve my own software to eliminate
any future issues.  No need to replay to reply to anything I post again
over 7 days old in the future.  Frankly I won't post again since I don't
have time to wait around three months for some idiotic response.  One thing
lacking is documentation and without documentation these open sourc
eprojects are worthless in the ends.

Thanks for your response, but all of the issues have been solved and I
promise not to bother you again.
